<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ModelImporter</class>
 <widget class="QWidget" name="ModelImporter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="1">
    <widget class="QLineEdit" name="imageDir"/>
   </item>
   <item row="3" column="1">
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>Batch convert</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="rootDir"/>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="modelDir"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Image directory</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Model directory</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Root directory</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>ModelImporter</receiver>
   <slot>onBatchConvert()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>170</x>
     <y>146</y>
    </hint>
    <hint type="destinationlabel">
     <x>181</x>
     <y>78</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onBatchConvert()</slot>
 </slots>
</ui>
